### ISSUE TITLE
Update social networks in the footer

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.org#489]
Currently we list GitHub, Twitter, YouTube in the footer. Instead I think we should have LinkedIn, GitHub, YouTube, Mastodon, Bluesky? Not sure about the order. cc @vossisboss @cnk

**Comment by vossisboss** [comment]
I agree with that @thibaudcolas. As for order, I say let's put them in whatever order the logos and links look nicest in.

**Comment by cnk** [comment]
GitHub and YouTube are in the footer currently. URLs for other options: 

- https://bsky.app/profile/wagtail.org
- https://fosstodon.org/@wagtail
- https://www.linkedin.com/company/wagtail-cms/

**Comment by suvanbanerjee** [comment]
@cnk I would like to take this issue

**Comment by cnk** [comment]
@suvanbanerjee We don't assign issues but would be happy for you work on it. 

Based on where I see references to the current social media icons, you will probably want to explore setting up the Django Pattern Library and then make modifications to the templates you see there. https://github.com/wagtail/wagtail.org/blob/4d670cea7fc8296cf9a92558d7a0583bdc0df5c7/docs/frontend/general-info.md

**Comment by thibaudcolas** [comment]
I’ve just manually removed Twitter from the footer as I don’t think we want to be associated with it at this time. Be great to update the footer more fully to showcase the other networks.

@suvanbanerjee are you still working on this or no?

**Comment by suvanbanerjee** [comment]
> @suvanbanerjee are you still working on this or no?

@thibaudcolas because of my exams this issue slipped really sorry for that I will complete this by this weekend

**Comment by thibaudcolas** [comment]
thanks, that’s completely fine! No worries if you don’t get to it

**Comment by shwetd19** [comment]
Hey @vossisboss @cnk  can you please review my PR at #497 
Pls let me know if any change is required, thanks!

**Comment by vossisboss** [comment]
@shwetd19 Thank you for letting us know about this and thank you SO much for contributing this PR. I wasn't able to fit it in this week, but I will do my best to have a look at your work next week.

It passed all the checks, so I ask that you please sit tight and have patience with me as I find time to test this.

Thank you again!!!
